### PR TITLE
#16: added tests for adapter notes

### LINKED ISSUE #16: Review AdapterNotes classes and add tests
AdapterNotes classes have missing unit tests. We should check them and add tests.

Moved from https://github.com/exasol/virtual-schemas/issues/277

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #16

### CODE REVIEW

File: src/main/java/com/exasol/adapter/adapternotes/ColumnAdapterNotes.java [hunk 1]
```diff
@@ -89,4 +91,19 @@ private static void checkKey(final JsonObject root, final String key, final Stri
                     + ". Please refresh the virtual schema");
         }
     }
+
+    @Override
+    public boolean equals(final Object o) {
```
Comment (redcatbear) [inline src/main/java/com/exasol/adapter/adapternotes/ColumnAdapterNotes.java]: rename `o` to `other`

File: src/test/java/com/exasol/adapter/adapternotes/ColumnAdapterNotesTest.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    @Test
+    void testGetTypeName() {
+        assertThat(columnAdapterNotes.getTypeName(), equalTo("DOUBLE"));
+    }
+
+    @Test
+    void testSerialize() {
+        assertThat(ColumnAdapterNotes.serialize(columnAdapterNotes),
+                equalTo("{\"jdbcDataType\":8,\"typeName\":\"DOUBLE\"}"));
```
Comment (redcatbear) [inline src/test/java/com/exasol/adapter/adapternotes/ColumnAdapterNotesTest.java]: Use JSONAssert.

File: src/main/java/com/exasol/adapter/adapternotes/ColumnAdapterNotes.java [hunk 3]
```diff
@@ -68,14 +70,14 @@ public static ColumnAdapterNotes deserialize(final String columnAdapterNotes, fi
             throws AdapterException {
```
Comment (redcatbear) [inline src/main/java/com/exasol/adapter/adapternotes/ColumnAdapterNotes.java]: This class needs refactoring. The column name is passed just to have something for the exception messages. Seems like unnecessary detail disclosure at this point.

But: not now. Please write an issue ticket.

File: src/main/java/com/exasol/adapter/adapternotes/ColumnAdapterNotes.java (hunk 1 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/adapternotes/ColumnAdapterNotes.java]: done

File: src/main/java/com/exasol/adapter/adapternotes/ColumnAdapterNotes.java (hunk 3 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/adapternotes/ColumnAdapterNotes.java]: created ticket

**Pull request merged**